src/Microsoft.AspNetCore.Hosting/WebHostBuilderExtensions.cs

```diff
@@ -92,5 +93,59 @@ public static IWebHostBuilder UseDefaultServiceProvider(this IWebHostBuilder hos
                 services.Replace(ServiceDescriptor.Singleton<IServiceProviderFactory<IServiceCollection>>(new DefaultServiceProviderFactory(options)));
             });
         }
+
+        /// <summary>
+        /// Runs a web application with the specified handler
+        /// </summary>
+        /// <param name="hostBuilder">The <see cref="IWebHostBuilder"/> to run.</param>
+        /// <param name="handler">A delegate that handles the request.</param>
+        public static IWebHost Run(this IWebHostBuilder hostBuilder, RequestDelegate handler)
```

Comment:
Why aren't these extensions named Start? the IWebHost.Run extensions all block but these call Start and return.